### ISSUE TITLE
Host to support CentOS, CentOS install the Paz ?

### ISSUE THREAD

**Issue by ytc301** [yldio/paz-web#17]
Host to support CentOS, CentOS install the Paz ?

**Comment by lukebond** [comment]
Hello @ytc301. Thanks for your interest in Paz!

This is possible, but not something that is on our roadmap at the moment. Some quick thoughts on what would be required to get Paz running on CentOS:
- it would need to be a version of CentOS that uses systemd. I believe this is CentOS 7? systemd is a dependency of Fleet, which is a dependency of Paz
- We use the cloud-config files to bootstrap our CoreOS hosts. You would need something similar that works for CentOS that achieves the same thing. Basically you need the stack described here in the "Anatomy of a Paz Host" diagram here: http://paz.readme.io/v1.0/docs/paz-architecture <- that's what our cloud-config does. There may be some things in the scripts in the cloud-config that are CoreOS dependant.

In summary: it's possible, not too complicated but probably time-consuming. If you want to try it then I'll be happy to answer any questions you have, but we have no plans to support CentOS and I think it would be unlikely it would ever be put onto our roadmap.

I hope that helps!

**Comment by lukebond** [comment]
Going to close this as it isn't on our roadmap. @ytc301 feel free to ask further questions and I'll help you.
